app/page.tsx

```diff
@@ -1,15 +1,32 @@
 "use client";
 
+import { auth0UserContext } from "@/types/userContext";
 import { useUser, withPageAuthRequired } from "@auth0/nextjs-auth0/client";
+import Layout from "./components/Layout";
+import Deliveries from "./components/Deliveries";
```

Comment:
Maybe let's export components as named exports, so that the import lines will be more easy to read. Also, IMO named exports lead to better codebase since the consumer don't have the freedom of naming it as they please.

Reply:
For now it's fine